src/common/dataHooks/useMySites.js

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from 'react-query'
+
+// FPCC
+import { MY_SITES } from 'common/constants'
+import api from 'services/api'
+import placeholder from 'images/cover-thumbnail.png'
+
+export default function useMySites() {
+  const userSitesResponse = useQuery([MY_SITES], () => api.site.mySites())
+  const formattedUserSitesData = userSitesResponse?.data?.map((site) => ({
+    uid: site?.id,
+    title: site?.title,
+    sitename: site?.slug,
+    visibility: site?.visibility?.toLowerCase(),
+    logoPath: site?.logo?.content || placeholder,
+    parentLanguageTitle: site?.language,
+    features: site?.features,
+  }))
+  return {
+    mySitesData: formattedUserSitesData,
```

Comment:
I'd be curious about Guy's thoughts here, but I wonder if the intention was for all the hooks to return a standard object that looks like `{ ...response, data: formattedData }`